mcux/mcux-sdk/drivers/lpuart/fsl_lpuart.h

```diff
@@ -650,9 +650,9 @@ uint32_t LPUART_GetEnabledInterrupts(LPUART_Type *base);
  * @param base LPUART peripheral base address.
  * @return LPUART data register addresses which are used both by the transmitter and receiver.
  */
-static inline uint32_t LPUART_GetDataRegisterAddress(LPUART_Type *base)
+static inline uintptr_t LPUART_GetDataRegisterAddress(LPUART_Type *base)
 {
-    return (uint32_t) & (base->DATA);
+    return (uintptr_t) & (base->DATA);
```

Comment:
Need to document this in the mcux/README in the patch list. Also, submit this back to the SDK team so we don't have to keep changing.